src/apis/edit.rs

```diff
@@ -25,9 +25,42 @@ pub struct EditResponse {
 }
 
 pub async fn edit(
-    req: actix_web::web::Json<EditRequest>,
+    req: actix_web::HttpRequest,
+    req_data: actix_web::web::Json<EditRequest>,
+    data: actix_web::web::Data<fpm::commands::serve::AppState>,
 ) -> actix_web::Result<actix_web::HttpResponse> {
-    match edit_worker(req.0).await {
+    let mut config = match fpm::Config::read(None, false).await {
+        Ok(config) => config,
+        Err(err) => {
+            return fpm::apis::error(
+                err.to_string(),
+                actix_web::http::StatusCode::INTERNAL_SERVER_ERROR,
+            )
+        }
+    };
+    config.set_identities(data.local_identities.clone());
+    config.current_document = Some(req_data.path.to_string());
```

Comment:
Mention here and possibly in Config struct that these are needed for processor. Are they only needed for processor?

Reply:
I think we should have config and some sort of "request context", later will keep request specific data, so maybe we will not have to construct Config all the time.